tooling-server-config/src/test/java/org/jboss/eap/qe/ts/common/server/config/ManagementClientAgainstStandaloneModeTest.java

```diff
@@ -0,0 +1,104 @@
+package org.jboss.eap.qe.ts.common.server.config;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.CliException;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+/**
+ * Test case to verify {@link BaseManagementClientFactory} features - together with Creaper implementations for both
+ * on-line and off-line management clients - against Wildfly running in standalone mode.
+ */
+public class ManagementClientAgainstStandaloneModeTest extends BaseManagementClientTestCase {
```

Comment:
This test coverage is nice, yet IMHO little bit extensive. Seems like duplication of Creaper tests to me.